example/domain/number-guessing-game/mongodb/spring/blocking/src/test/java/org/occurrent/example/domain/numberguessinggame/mongodb/spring/blocking/TestBootstrap.java

```diff
@@ -31,7 +31,7 @@ public class TestBootstrap {
     @Bean
     @ServiceConnection
     public static MongoDBContainer mongoDBContainer() {
-        return new MongoDBContainer("mongo:7.0.9");
```

Comment:
@johanhaleby please note that I've downgraded version in the example, but I don't think it's a problem. I would like to highlight, that upgrading MongoDB in tests from `4.2.8` to the latest `8.0.13` will lead to integration test failures.

Reply:
Interesting, I didn't know this. We're using 8.x in production though and to my knowledge we haven't ran into any problems because of this.